packages/vitest/src/node/logger.ts

```diff
@@ -231,52 +228,28 @@ export class Logger {
       const origin = resolvedUrls?.local[0] ?? resolvedUrls?.network[0]
       const provider = project.browser.provider.name
       const providerString = provider === 'preview' ? '' : ` by ${provider}`
-      this.log(
-        c.dim(
-          c.green(
-            `     ${output} Browser runner started${providerString} at ${new URL('/', origin)}`,
-          ),
-        ),
-      )
+
+      this.log(PAD + c.dim(c.green(`${output} Browser runner started${providerString} at ${new URL('/', origin)}`)))
     })
 
     if (this.ctx.config.ui) {
-      this.log(
-        c.dim(
-          c.green(
-            `      UI started at http://${
-              this.ctx.config.api?.host || 'localhost'
-            }:${c.bold(`${this.ctx.server.config.server.port}`)}${
-              this.ctx.config.uiBase
-            }`,
-          ),
-        ),
-      )
+      this.log(PAD + c.dim(c.green(`UI started at http://${this.ctx.config.api?.host || 'localhost'}:${c.bold(`${this.ctx.server.config.server.port}`)}${this.ctx.config.uiBase}`)))
```

Comment:
can we make temporary variables for evert url part? this seems too overloaded

Reply:
Sure! This indeed is a bit too compact atm. I'll add separate variables here.